packages/mix/lib/src/modifiers/scroll_view_widget_modifier.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/widgets.dart';
+import 'package:mix/mix.dart';
+import 'package:mix_annotations/mix_annotations.dart';
+
+part 'scroll_view_widget_modifier.g.dart';
+
+@MixableSpec(skipUtility: true, withLerp: false)
+final class ScrollViewModifierSpec
```

Comment:
We acknowledge that there are numerous customizations possible with the `SingleChildScrollView`. However, supporting all of them would compromise the architecture. The `Attributes` were intended for visual styles, so if we incorporate properties like controllers, it would begin to manage behavior that is not its intended function.

Reply:
based on this comment above, this modifier shouldn't be a complete API to become widgets scrollable. It's more like a simple way to handle simple cases.